_src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown

```diff
@@ -101,7 +101,7 @@ Finally, to build an install threshold for today's date, we take a timespan of d
 
 #Conclusion
 
-The methods here exposed are a first iteration for fraud detection and classification for Click Spamming. Note here that we rely on the assumption that for all apps, there are periods free of fraudulent behavior. These periods are then used to calculate our final threshold which is robust to data that is contaminated. We are confident on this assumption since we have seen that fraudulent behavior from publishers will only last, at most, for a few days. Thus using at least a week of data is enough to identify cases of fraudulent behavior.
+The methods here exposed are a first iteration for fraud detection and classification for Click Spamming. Note here that we rely on the assumption that for all apps there are periods free of fraudulent behavior. These periods are then used to calculate our final threshold which is robust to data that is contaminated. We are confident on this assumption since we have seen that fraudulent behavior from publishers will only last, at most, for a few days. Thus using at least a week of data is enough to identify cases of fraudulent behavior.
```

Comment:
We can endlessly revise the nuances of the English language. I think that we can start wrapping up if you guys approve.